src/components/TreeReactFlow/Flavor.ts

```diff
@@ -522,3 +529,72 @@ export const boxFlavorBackground = (flavor: NodeFlavorBoxBody): string => {
       return "bg-yellow-primary";
   }
 };
+
+/** What sort of node does this flavor correspond to?
+ * Note that the backend could in principal tell us this independently of flavors,
+ * since it comes down to whether the node ultimately comes from an `Expr`, `Type` or `Kind`.
+ */
+export const isTypeLevel = (flavor: NodeFlavor): "term" | "type" | "kind" => {
```

Comment:
Is there a good name for "type or term or kind"? The existing naming here was just about okay when we weren't considering kinds, but now it's misleading.

Reply:
I would probably use "sort". It is used this way in Pure Type Systems.